app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -6,6 +6,7 @@ class ManageIQ::Providers::Amazon::NetworkManager::RefreshParser
   def initialize(ems, options = nil)
     @ems        = ems
     @aws_ec2    = ems.connect
+    @aws_elb    = @ems.connect(:service => :ElasticLoadBalancing)
```

Comment:
for consistency I would either use `ems.connect` or change above to `@ems.connect`